### PR TITLE
Add biomass partitioning section to BioCro vignette

### PR DESCRIPTION
**Pull request by KristinaRiemer**
This includes #1, only need to look at biomass partition commit.

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
@kimberlyh66 if you want to look through the three vignettes (biocro_095.pdf, biocro_in_pecan.pdf, ed2_in_pecan.pdf) and try to run their contents, that would be great! The first is going through the high level functionality of the BioCro model by itself, the second is for running BioCro using PEcAn, and the third is for running the ED2 model in PEcAn. Let me know if you have any suggestions or questions, especially if any of it needs more explanation.

**Comment by kimberlyh66** [comment]
I am getting the following error when trying to start up the pecan virtual machine. Have you seen this error before?

pecan_virtual_machine_error

File: ED2/ed2_in_pecan.Rmd [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+author: "Kristina Riemer"
+date: "3/25/2019"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+## Run model
+
+The Ecosystem Demography ([ED2](https://github.com/EDmodel/ED2)) model was run using the [PEcAn VM in Virtualbox](https://pecanproject.github.io/pecan-documentation/develop/GettingStarted.html#installing-and-running-pecan). The following settings were used:
```
Comment (kimberlyh66) [inline ED2/ed2_in_pecan.Rmd]: Where can you check/set up these settings using the pecan virtual machine?

Comment (kimberlyh66) [inline ED2/ed2_in_pecan.Rmd]: I am not sure how to set up this ED2 model run using the VM

**Comment by dlebauer** [comment]
Haven't seen it but try installing the extension pack ..

Is there a reason to test this on the VM rather than Welch?

_______________________
David LeBauer
Director of Data Sciences
Arizona Experiment Station
THE UNIVERSITY OF ARIZONA

Bioscience Research Labs, 207
1230 N Cherry Ave | Tucson, AZ 85721
Office: 520-621-4381
dlebauer@email.arizona.edu

(sent from my phone - please pardon brevity and typos)


________________________________

**Comment by KristinaRiemer** [comment]
@dlebauer I'm planning on doing this in Welsch now that I know how to deal with the missing FQDN problem, had to do it in the VM just to get model runs for both models to work. I don't know if we should include instructions for both the VM and Welsch? 

@kimberlyh66 installing and getting the VM to work has definitely been difficult. Like David said, you should probably try downloading the extension pack from the Virtualbox website and add it by going to preferences -> extensions within the Virtualbox application. This info should be added to the Pecan documentation.

**Comment by KristinaRiemer** [comment]
@kimberlyh66 I think this most recent commit addresses all of your comments. Let me know about additional or followup suggestions.

**Comment by kimberlyh66** [comment]
@KristinaRiemer I think the vignettes look good! I do not have any more suggestions. 

I was able to setup the ED2 model run in the pecan vm using the new instructions. I think they are easy to follow and clear.

**Comment by KristinaRiemer** [comment]
@kimberlyh66 great, thanks for your work! 

@dlebauer I'm going to merge this for now and then we can talk about what direction we want to head in with these vignettes, in terms of VM/Welsch and using Danforth data.

**Pull request merged**